esp/services/ws_sasha/ws_sashaservice.cpp

```diff
@@ -20,10 +20,192 @@
 #endif
 
 #include "ws_sashaservice.hpp"
-#include "jlib.hpp"
+#include "sashacli.hpp"
 
 using namespace ws_sasha;
 
+constexpr const char* FEATURE_URL = "SashaAccess";
+
+ISashaCommand* CWSSashaEx::setSashaCommand(SashaCommandAction action, bool dfu, SocketEndpoint& serverep)
```

Comment:
I did not use a createXX method because the createSashaCommand() (defined in sacmd.hpp) is called in line 34. I may rename this one to createAndSetSashaCommand.